lib/ansible/plugins/doc_fragments/action_common_attributes.py

```diff
@@ -10,7 +10,7 @@ class ModuleDocFragment(object):
     DOCUMENTATION = r'''
 attributes:
     check_mode:
-      description: Can run in check_mode and return changed status prediction without modifying target
+      description: Full support means that it can be run in check_mode and return changed status prediction without modifying target
```

Comment:
i think the confusion is on what happens when not supported:
`Can run in check_mode and return changed status prediction without modifying target, if not supported the action will be skipped.`

Reply:
Thanks you @bcoca, You're rigth. I will update my PR according to your proposal